### PR TITLE
Switch publishing to `s01.oss.sonatype.org`

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,7 +15,7 @@ lazy val core = project
 
 lazy val docs = project.in(file("site")).enablePlugins(Http4sOrgSitePlugin)
 
-ThisBuild / tlBaseVersion := "0.11"
+ThisBuild / tlBaseVersion := "0.12"
```
Comment (armanbilge) [inline build.sbt:18]: This is a breaking change.

**Comment by rossabaker** [comment]
Okay, I'm going to publish from organization secrets. 🤞

**Comment by armanbilge** [comment]
The snapshot publish just failed. Did you delete the secrets from this repository?

**Comment by armanbilge** [comment]
Oh, but the tag seems to have worked! 😁

**Comment by rossabaker** [comment]
I think I merged, then cleared out the old secrets, then tagged... though they should have had rights on the new host too.

**Comment by armanbilge** [comment]
They wouldn't have I think. `oss.sonatype.org` and `s01.oss.sonatype.org` have independent credential systems.

**Comment by rossabaker** [comment]
From the Sonatype ticket:

>  NOTE: all users who previously had access to this groupId should now be able to access it on the new host AND I've granted http4s-bot permissions as well, but ONLY on the new host.

:shrug: It doesn't really matter, because the one we want to use works on the host we want to use it on.

**Comment by armanbilge** [comment]
It's confusing. The _users_ have permissions, but the credentials do not. I ran into this when you got me publishing rights for Typelevel and my s01 credentials didn't work. I had to log-in to the old system and generate a separate pair of credentials.

**Comment by armanbilge** [comment]
Oh, apologies, I'm confusing user credentials vs user tokens. I exclusively use the latter. Also from the ticket:

> Also, keep in mind that if you are using Tokens instead of username/credentials to publish you will need to generate new tokens and replace those in your current setup from here

**Comment by rossabaker** [comment]
Oh, I usually set up tokens.  It was almost certainly that.

**Pull request merged**